js/forum/src/components/SignUpModal.js

```diff
@@ -60,6 +60,10 @@ export default class SignUpModal extends Modal {
     ];
   }
 
+  fieldCantBeEdited(field) {
```

Comment:
Could we please change this to `isProvided(field)`?

Reply:
I agree to change this, but isn't `isProvided` a bit vague about its purpose as well ?

Reply:
`isProvided` reads nicely for me. "disable this field if its value is provided"

`isIdentifier` could also work, "disable this field if it's an identification field"